habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -456,23 +496,32 @@ def step(self, action: Union[str, int]) -> Observations:
 
         self._update_markers()
 
-        if self._should_render_debug:
+        if self._debug_render:
+            rom = self.get_rigid_object_manager()
             self._try_acquire_context()
-            for obj_handle, _ in self.ep_info["targets"].items():
+            # Don't draw bounding boxes over target objects.
+            for obj_handle, _ in self._targets.items():
```

Comment:
Why not draw bounding boxes over target objects? Was this comment intentional? It is important to draw the bounding boxes to distuingish from target positions and real objects. This is not important for Pick but it is for other tasks.

Reply:
I think you are the one who added this comment : https://github.com/facebookresearch/habitat-lab/blame/hab_suite_dev/habitat/tasks/rearrange/rearrange_sim.py#L502 I am just porting it to main. If this code needs to change, can you advise?